README.md

```diff
@@ -6,10 +6,12 @@ This repository contains examples showing how to consume the PowerDMS File Picke
 
 The File Picker is implemented as an IFrame, with the contents hosted at `https://filepicker.powerdms.com`.
 
-There is a small initialization script and stylesheet that must be referenced on the consuming page to create and cleanup the IFrame. The initialization script will add a function called `initializePowerDmsFilePicker` to the `window` which will create the IFrame. The consuming page must have a button that will call this function.
+There is a small [initialization script and stylesheet](https://github.com/PowerDMS/PowerDMS/tree/develop/src/clients/file-picker/initializer) that must be referenced on the consuming page to create and cleanup the IFrame. The initialization script will add a function called `initializePowerDmsFilePicker` to the `window` which will create the IFrame. The consuming page must have a button that will call this function.
 
 The function `initializePowerDmsFilePicker` accepts a configuration object. One of the properties of this object is a function `onFileSelected` which is called when the user selects a PowerDMS document. When `onFileSelected` is called, the File Picker will pass information back to the consuming page, including the URL to use to get the file contents from the PowerDMS API. (See [link below](#See-also) for more details on the API.)
 
+To see sample implementation code of the PowerDMS File Picker in further detail, visit our [Mock Partner Site documentation](https://github.com/PowerDMS/PowerDMS/tree/develop/src/clients/file-picker/mock-partner-site). 

```

Comment:
🔧 Customers will not have access to our version of the mock partner site. This repo is effectively a simpler customer-facing copy of the mock partner site. You could link to `https://github.com/PowerDMS/File-Picker-Sample-Implementations/tree/master/aspnetcore/FilePickerSample` though